tensorboard/plugins/interactive_inference/interactive_inference_plugin.py

```diff
@@ -55,6 +55,7 @@ class InteractiveInferencePlugin(base_plugin.TBPlugin):
   examples = []
   updated_example_indices = set()
   sprite = None
+  example_class = tf.train.Example
```

Comment:
Unsure but first character of the class name may need to be capitalized instead of snake_case

Reply:
true, changed to adhere to style guide

Reply:
Seems like a conflict between style for member variables (snake) and class names (CamelCase), of which this is both. Will keep as this to make it clear its a member variable, since the name documents that it is a class, if that is ok